client/components/mma/cancel/contributions/ContributionsCancellationReasons.tsx

```diff
@@ -104,9 +104,19 @@ export const contributionsCancellationReasons: CancellationReason[] = [
 		linkLabel: 'I no longer read the Guardian enough to justify my support',
 		alternateFeedbackIntro: standardAlternateFeedbackIntro,
 	},
+];
+
+export const otherCancellationReason: CancellationReason[] = [
 	{
 		reasonId: 'mma_other',
 		linkLabel: 'Another reason (please specify)',
 		alternateFeedbackIntro: standardAlternateFeedbackIntro,
 	},
 ];
+
+const shuffledArray = contributionsCancellationReasons.sort(
+	() => 0.5 - Math.random(),
+);
+
+export const shuffledContributionsCancellationReasons: CancellationReason[] =
+	shuffledArray.concat(otherCancellationReason);
```

Comment:
could be done with a spread operation, ideally with the value of `otherCancellationReason` inlined (if not reused, and if reused then it needn't be wrapped in array), something like... 
```suggestion
	[...shuffledArray , otherCancellationReason];
```

Reply:
Great suggestion!